misc/git-hook/install.ps1

```diff
@@ -0,0 +1,40 @@
+param([switch]$all, [switch]$clear)
+
+$gitDir = git rev-parse --git-common-dir
+
+if (!(test-path $gitDir)) 
+{
+   $gitDir = git rev-parse --git-dir
+}
+
+$gitDir = (resolve-path ($gitDir)).Path
+
+if (!$gitDir) {
+    throw ".git - Directory not found"
+}
+
+$gitHooksDir = join-path $gitDir "hooks"
+
+if (!(test-path $gitHooksDir)) {
+    throw "$gitHooksDir not present"
+}
+
+if($clear) {
+    write-host "cleaning old hooks dir: $githooksdir"
+    Remove-Item (join-path $githooksdir "*") -force -recurse

```

Comment:
Also clear removes all the 'default sample git hooks'. Might be wise to not remove those